tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -134,13 +136,13 @@ DOCKER_RUN_PARAMETER="--volume $SOURCE_ROOT:/onnxruntime_src \
                       --volume /data/onnx:/data/onnx:ro \
                       --volume $HOME/.cache/onnxruntime:/home/onnxruntimedev/.cache/onnxruntime \
                       --volume $HOME/.onnx:/home/onnxruntimedev/.onnx"
-if [ $BUILD_DEVICE = "openvino" ] && [[ $BUILD_EXTR_PAR == *"--use_openvino GPU_FP"* ]]; then
+if [ "$BUILD_DEVICE" = "openvino" ] && [[ $BUILD_EXTR_PAR == *"--use_openvino GPU_FP"* ]]; then
     DOCKER_RUN_PARAMETER="$DOCKER_RUN_PARAMETER --device /dev/dri:/dev/dri"
 fi
 # Though this command has a yocto version argument, none of our ci build pipelines use yocto.
-$DOCKER_CMD run $RUNTIME --rm $DOCKER_RUN_PARAMETER \
+$DOCKER_CMD run "$RUNTIME" --rm "$DOCKER_RUN_PARAMETER" \
```

Comment:
Please remove the unnecessary double quotes around variables in the Docker command to prevent extra spaces from being introduced